modules/nixos/monitoring/telegraf.nix

```diff
@@ -0,0 +1,43 @@
+{
+  services.telegraf.extraConfig.inputs = {
```

Comment:
@Mic92 I was going to move this stuff first so the following PR can focus on prometheus, etc.